core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/ScoreOperationChainHandler.java

```diff
@@ -66,7 +66,11 @@ public int getChainScore(final OperationChain<?> opChain, final User user) {
 
         if (null != opChain) {
             for (final Operation operation : opChain.getOperations()) {
-                chainScore += authorise(operation);
+                if (operation instanceof OperationChain) {
+                    chainScore += getChainScore((OperationChain<?>) operation, user);
```

Comment:
Please could you add some tests to check nested operations are handled correctly.